src/NACHAScan.java

```diff
@@ -4,9 +4,42 @@
 // for protected cards.
 //
 
+public class NACHAScan{
 
+    public static boolean isNumeric(String strNum) {
+        if (strNum == null) {
+            return false;
+        }
+        try {
+            double d = Double.parseDouble(strNum);
```

Comment:
@theblasher  This will make our solution perform poorly due to the overhead of exception handling.

use the Double.tryParse() instead.  Eliminates the need for try catch and you can just check the return value from tryParse()